oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/DocumentNodeStoreServiceConfiguration.java

```diff
@@ -73,6 +73,12 @@
      */
     private static final String FWK_PROP_SO_KEEP_ALIVE = "oak.mongo.socketKeepAlive";
 
+    /**
+     * Name of framework property to configure socket timeout for lease update
+     * operations on MongoDB.
+     */
+    private static final String FWK_PROP_LEASE_SO_TIMEOUT = "oak.mongo.leaseSocketTimeout";
```

Comment:
To @reschke 's comment re naming/generalising : what I noticed is that the property key has `mongo` in it (to make it clear, it is only for mongo) - but this field does not. Perhaps it could be renamed to sth like `FWK_PROP_MONGO_LEASE_SO_TIMEOUT` (and the same would then also apply to other places/methods that use leaseSoTimeout and not specifically call it out as belonging to mongo

Reply:
Not sure I understand. Do you suggest turning the configuration `leaseSocketTimeout` into a generic one that also applies to RDB or rather making it fully MongoDB specific?

Reply:
I meant making it fully MongoDB specific

Reply:
Ack; I misread the original ticket; primary/secondary config does not apply to RDB.

Reply:
Done with https://github.com/apache/jackrabbit-oak/pull/283/commits/da55aa3195fe63a249b19ebc86e3e7509cb92439